src/test/java/uk/gov/hmcts/reform/draftstore/service/draftservice/DeleteTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.draftstore.service.draftservice;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.draftstore.exception.AuthorizationException;
+import uk.gov.hmcts.reform.draftstore.service.UserAndService;
+
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DeleteTest extends BaseTest {
+
+    @Test
+    public void should_throw_an_exception_when_trying_to_remove_somebody_elses_draft() throws Exception {
```

Comment:
We could add a case for trying to remove the same person's draft for a different service and a case when we successfully delete an existing draft (we could verify repository call here).